app/workers/reactions/bust_reactable_cache_worker.rb

```diff
@@ -14,6 +14,13 @@ def perform(reaction_id)
       case reaction.reactable_type
       when "Article"
         cache_bust.call("/reactions?article_id=#{reaction.reactable_id}")
+        article = reaction.reactable
+
+        # We only want to bust on the creation or deletion of the "first" reaction.
+        # This is logically called *after* creation, but *before* deletion. So "1" is correct in each case.
+        if article && article.reactions.public_category.size == 1
```

Comment:
This is a background worker, so it can afford to be less efficient, but we're already checking existence on line 9, what would you think about cutting down on queries here? 

```suggestion
        if Reaction.for_articles([reaction.reactable_id]).public_category.size == 1
```